app/models/manageiq/providers/ansible_tower/automation_manager/machine_credential.rb

```diff
@@ -1,2 +1,66 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential < ManageIQ::Providers::AutomationManager::Authentication
+  FIELDS = {
+    :description => {
+      :type       => :string,
+      :required   => false,
+      :label      => _('Description'),
+      :help_text  => _('Optional description of this credential.'),
+      :encrypted  => false
```

Comment:
I think you can leave out the encrypted value, since most every field will be false.  The call can assume that a nil/no value is false.  @bdunne Thoughts?